sdk/textanalytics/ai-text-analytics/test/textAnalyticsClient.spec.ts

```diff
@@ -597,4 +603,96 @@ describe("[AAD] TextAnalyticsClient", function() {
       }
     });
   });
+
+  describe("#String encoding", () => {
+    it("emoji", async () => {
+      const [result] = await client.recognizePiiEntities([
```

Comment:
@iscai-msft
I do get the same offset value in these tests like JS. But I found python is getting different values. 
https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/textanalytics/azure-ai-textanalytics/tests/test_encoding.py#L28

Reply:
@mssfang yes because python is using a different encoding than us.